docs/scos/user/intro-to-spryker/releases/release-notes/security-release-notes-202312.1.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Security release notes 202312.1
+description: Security release notes for 202312.1
+last_updated: Dec 7, 2023
+template: concept-topic-template
+---
+
+The following information pertains to security-related issues that have been recently resolved. All issues are listed by description and affected modules.
+
+If you need any additional support with this content, [contact our support](https://support.spryker.com/). If you found a new security vulnerability, inform us through [security@spryker.com](mailto:security@spryker.com).
+
+## Token validation issue
+
+Backend API access tokens are not validated. An attacker is able to request as many access tokens as he likes. Each requested token is valid and not expired.
```

Comment:
`Backend API access tokens are not validated` sounds to scary IMO :)
I would rephrase it to smth like `Glue Backend API has no expiration validation for token` or smth like this